example/package.json

```diff
@@ -17,6 +17,7 @@
   "devDependencies": {
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
+    "browserify": "^15.2.0",
```

Comment:
Why did you add this dependency here?

Reply:
The tests were failing for me without it after a fresh install. But I see now it should be installed globally...my global modules got messed up after a node reinstall. I'll remove it :)